src/app/components/popup-choose-the-time/time/time.component.html

```diff
@@ -0,0 +1,7 @@
+<label>
+  from
+  <input type="time" />
+  to
+  <input type="time" />
+  <button class="btn unfilled-btn btn-minus" (click)="removeObject()">-</button>
+</label>
```

Comment:
please, separate label by two for each input

Reply:
fix